index.bs

```diff
@@ -63,43 +69,27 @@ files and directories they've been given access to, allowing the web apps to
 later regain access to the same content without requiring the user to select the
 same file again.
 
+This API extends the API in [[FS]], which specifies an [=origin private file system=]
+which websites can get access to without having to first prompt the user for access.
```

Comment:
I'm not sure if this whole description of what the other spec defines still makes sense in this spec? I'd focus more on what this spec adds to it, rather than what is already in the other fs spec.